pom.xml

```diff
@@ -35,11 +36,11 @@
     <kotlin.version>1.3.61</kotlin.version>
     <kotlin-logging.version>1.7.7</kotlin-logging.version>
     <jackson-kotlin.version>2.10.0</jackson-kotlin.version>
-    <mockk.version>1.9.3</mockk.version>
+    <mockk.version>1.10.0</mockk.version>
     <junit.jupiter.version>5.5.2</junit.jupiter.version>
     <slf4j.version>1.7.30</slf4j.version>
     <log4j.version>2.13.3</log4j.version>
-    <dokka.version>0.10.1</dokka.version>
+    <dokka.version>1.4.0</dokka.version>
```

Comment:
Won't this also bump Kotlin version to 1.4?

Reply:
Oh, I just did push the extension to 1.4, as a follow up of dependabot, which I figured to be the right thing to do. If this was wrong, please correct me on it.

Reply:
Kotlin 1.3 -> 1.4 is not a significant change code wise, so everything should be fine. That does mean that the extension will be dependent on Kotlin 1.4 from now on.